test/Mono.Linker.Tests.Cases/Libraries/CopyAssemblyWithMainEntryRoot.cs

```diff
@@ -0,0 +1,29 @@
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.Libraries
+{
+#if !NETCOREAPP
+	[IgnoreTestCase ("Correctly handled by illink only")]
+#endif
+	[Kept]
+	[KeptMember (".ctor()")]
+	[SetupLinkerAction ("copyused", "test")]
+	public class CopyAssemblyWithMainEntryRoot
```

Comment:
Should the name be `CopyUsedASsemblyWithMainEntryRoot` ? since the test runs it with "copyused" action.

Reply:
Fixed